Backpack/src/main/java/net/skyscanner/backpack/horisontalnav/BpkHorizontalNav.kt

```diff
@@ -223,9 +217,7 @@ open class BpkHorizontalNav @JvmOverloads constructor(
         it.setBackground(ColorStateList.valueOf(Color.TRANSPARENT), tabTextColors!!)
         it.setTextColor(tabTextColors)
       }
-      findViewById<TextView>(android.R.id.text1)?.let {
-        it.setTextColor(tabTextColors)
-      }
+      findViewById<TextView>(android.R.id.text1)?.setTextColor(tabTextColors)
```

Comment:
just cleaned up a warning - the let isn't needed as we're just setting one property